pkg/cmd/get_clustercompliancereport.go

```diff
@@ -50,7 +52,11 @@ func NewGetClusterComplianceReportsCmd(executable string, cf *genericclioptions.
 				return err
 			}
 			// generate compliance and compliance failure detail reports
-			complianceMgr := compliance.NewMgr(kubeClient, logger)
+			operatorConfig, err := etc.GetOperatorConfig()
```

Comment:
The CLI depends on the struct that's meant for the Operator and introduced the coupling between Go packages.